keyboards/xbows/ranger/rules.mk

```diff
@@ -13,7 +13,7 @@ EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
-SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
```

Comment:
Any reason for this?

Reply:
My keyboard always wakes up when my computer is asleep, this seems to be one way, and I'm looking for the optimal solution.

Reply:
I don't think this feature does what you think it does. It essentially pulses the Caps Lock LED when the host PC is in sleep mode, however this board has no Caps Lock LED defined, so it won't do anything.

Reply:
OK